.github/workflows/prbot.yml

```diff
@@ -113,6 +113,24 @@ jobs:
       - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4
       - uses: ./.github/actions/setup
       - run: npm run package-compatibility
+  typedoc:
+    name: 'Build typedoc'
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@91182cccc01eb5e619899d80e4e971d6181294a7 # v2.10.1
+        with:
+          egress-policy: audit
+
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4
+      - uses: ./.github/actions/setup
+      - name: 'Build headless typedoc'
+        run: npm run build:typedoc
+        working-directory: packages/headless
+      # - name: 'Build headless-react typedoc'
+      #   run: npm run build:typedoc
+      #   working-directory: packages/headless-react
```

Comment:
A addition to the prbot to make sure the thing still builds. We don't want it to fail during the release workflow lol. Also we don't want it to run at every build because it takes significant time (over 50 seconds).